File: javascript-closure-angular/API/Client/TypesIpfsPinStatusResponse.js
goog.provide('API.Client.types.IpfsPinStatusResponse');

/**
 * @record
 */
API.Client.TypesIpfsPinStatusResponse = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.TypesIpfsPinStatusResponse.prototype.created;

/**
 * @type {!Array<!string>}
 * @export
 */
API.Client.TypesIpfsPinStatusResponse.prototype.delegates;

/**
 * @type {!API.Client.Object}
 * @export
 */
API.Client.TypesIpfsPinStatusResponse.prototype.info;

/**
 * @type {!API.Client.types.IpfsPin}
 * @export
 */
API.Client.TypesIpfsPinStatusResponse.prototype.pin;

/**
 * @type {!string}
 * @export
 */
API.Client.TypesIpfsPinStatusResponse.prototype.requestid;

/**
 * @type {!string}
 * @export
 */
API.Client.TypesIpfsPinStatusResponse.prototype.status;

File: scala-akka-http-server/src/main/scala/io/swagger/server/model/Types.IpfsPinStatusResponse.scala
package io.swagger.server.model


/**
 * @param content 
 * @param created 
 * @param delegates 
 * @param info 
 * @param pin 
 * @param requestid 
 * @param status 
 */
case class Types.IpfsPinStatusResponse (
  content: Option[util.Content],
  created: Option[String],
  delegates: Option[List[String]],
  info: Option[Map[String, Object]],
  pin: Option[types.IpfsPin],
  requestid: Option[String],
  status: Option[types.PinningStatus]
)

